### PR TITLE
WASM client: add Faraday

### PR DESCRIPTION
**Pull request by guggero**
Adds the JSON/WASM stubs from the `frdrpc` package to allow calling Faraday functionality in LiT.

Has to be tested against LiT [PR #358](https://github.com/lightninglabs/lightning-terminal/pull/358)! The latest tagged version doesn't work because it didn't update to the LNC Handshake v2 yet.

### CODE REVIEW

**Review by ellemouton — APPROVED** [review]
Niiiiice 🔥

**Review by kaloudis — APPROVED** [review]

**Comment by guggero** [comment]
The race unit test fails because of a race condition in `lnd` that was fixed. We can't update to that version just yet, we want to wait for `lnd 0.15` to be tagged and https://github.com/lightninglabs/lightning-node-connect/pull/36 to be merged.

@kaloudis I think we should discuss the timing of this and similar changes offline.

File: go.mod [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
 	google.golang.org/grpc v1.39.0
 	google.golang.org/protobuf v1.27.1
 	nhooyr.io/websocket v1.8.7
 )
 
 replace git.schwanenlied.me/yawning/bsaes.git => github.com/Yawning/bsaes v0.0.0-20180720073208-c0276d75487e
 
-replace github.com/lightninglabs/lightning-node-connect/hashmailrpc => ./hashmailrpc
+replace github.com/lightninglabs/lndclient => github.com/lightninglabs/lndclient v0.14.2-3
```
Comment (ellemouton) [inline go.mod:22]: what is the reason for these `replace` directives if they are not referring to something in the `require` list? do they replace dependencies within one of the dependencies in the `require` list?

Comment (guggero) [reply go.mod:22]: It's a transitive dependency we pull in through `aperture`. But `aperture v0.1.17-beta` still uses `lndclient v0.12.0-9` which isn't compile time compatible with `lnd v0.14.3-beta` unfortunately (which I think is an issue in the `lndclient v0.12.x` branch).

**Comment by guggero** [comment]
I've merged #36 and bumped everything to `lnd v0.15.0-beta.rc3` which fixed the unit tests.
Going to merge this as well. If we need anything backward compatible (e.g. with handshake v2 not yet added), we can always create a branch for this.

**Pull request merged**